R/check_result.R

```diff
@@ -62,6 +64,6 @@ check_result <- function(
 
   return(graded(
     correct = final_result$correct,
-    message = message
+    message = final_result$message
```

Comment:
Why not keep it as `message`?

This value is created from the `final_result$message` in _new line 60_.   By using the glue statement, encouragement and praise can be added in addition to the passing / failure message provided by `final_result`.

Reply:
@chendaniely

Reply:
see comment: https://github.com/rstudio-education/grader/pull/21/files#r300204016